frontend/src/components/program/ScheduleEntries.vue

```diff
@@ -56,15 +61,23 @@ export default {
   },
   computed: {
     scheduleEntries() {
-      // TODO for SideBar, add filtering for the current day, now that the API supports it
-      return this.period().scheduleEntries()
+      if (this.day) {
+        return this.day().scheduleEntries()
+      } else {
+        return this.period().scheduleEntries()
+      }
     },
   },
   mounted() {
-    this.period().scheduleEntries().$reload()
-    this.period().camp().activities().$reload()
+    if (this.day) {
+      this.day().scheduleEntries().$reload()
+      // TODO which elements need a reload here and how?
```

Comment:
Do you want to keep that here or discuss it?

Reply:
Looking to discuss that before merging. That's why I went with a draft PR instead of a normal one.